app-text/podofo/podofo-0.10.1.ebuild

```
@@ -52,5 +55,23 @@ src_configure() {
 		$(cmake_use_find_package png PNG)
 		$(cmake_use_find_package fontconfig Fontconfig)
 	)
+
+	if test-flags-CXX -ffp-contract=off; then
+		# some optimizations cause testsuite failures which may indicate
+		# unsoundness with contraction. Be cautious for now. Reported
+		# upstream as https://github.com/podofo/podofo/issues/103
+		append-cxxflags -ffp-contract=off
+	fi
 	cmake_src_configure
 }
+
+src_test() {
+	local CMAKE_SKIP_TESTS=(
+		$(usex png '' 'TestImage3')
```

fwiw, you can do:
```suggestion
		$(usev !png 'TestImage3')
```
... and so on

---

From the devmanual:



`usev flagname` | As `use`, echoes `flagname` upon success.
-- | --

Now granted I don't actually know what that sentence means, but it doesn't imply to me that you can give it a second arg. So I grepped in /usr/lib/portage/python3.11 and it appears to be an EAPI=8 improvement.

Conclusion: this page is lacking in explanation. best_version / have_version are the *only* things there that specify "since EAPI=7 you can do $thing".

---

Yeah:
* https://mgorny.pl/articles/the-ultimate-guide-to-eapi-8.html#usev-now-accepts-a-second-argument
* https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-133002r20

devmanual needs updating clearly, which I'll file a bug for